docs/DocGenerator/config.yaml

```diff
@@ -6,8 +6,6 @@ Include paths:
   - wazuh_db:
     path: "../../tests/integration/test_wazuh_db"
     recursive: false
-  - vulnerability_detector:
```

Comment:
I think we should use the same separator. Underscore instead of space in all keys.

Reply:
Did you refer to the keys for other items in the config?
I think using spaces is more readable as it isn't required to be a unique word.
In this case, wazuh_db or vulnerability_detector refers to a Wazuh module.